test/jdk/java/util/Random/RandomTestCoverage.java

```diff
@@ -171,8 +171,25 @@ static void coverFactory(RandomGeneratorFactory factory) {
         boolean isSplittable = factory.isSplittable();
 
         coverRandomGenerator(factory.create());
-        coverRandomGenerator(factory.create(12345L));
-        coverRandomGenerator(factory.create(new byte[] {1, 2, 3, 4, 5, 6, 7, 8}));
+        RandomGenerator rng;
+
+        rng = null;
+        try {
+            rng = factory.create(12345L);
+        } catch (UnsupportedOperationException ignore) {
+        }
+        if (rng != null) {
+            coverRandomGenerator(rng);
+        }
+
+        rng = null;
+        try {
+            rng = factory.create(new byte[] {1, 2, 3, 4, 5, 6, 7, 8});
+        } catch (UnsupportedOperationException ignore) {
+        }
+        if (rng != null) {
+            coverRandomGenerator(rng);
+        }
     }
 
     static void coverDefaults() {
```

Comment:
This test method appears to test the calls to `getDefault()` methods on `RandomGeneratorFactory` and `RandomGenerator` classes, but it isn't being called in the test. We should call this method from `main()` to have test coverage for those methods.

Reply:
Right